### PR TITLE
gtk4: Add sample/examples/menubar.rb

### PR DESCRIPTION
**Pull request by SamiSieranoja**
Fix GH-1581

### CODE REVIEW

File: gtk4/sample/examples/menubar.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  window.set_application(app)
+
+  act_quit = Gio::SimpleAction.new("quit")
+  app.add_action(act_quit)
+  act_quit.signal_connect "activate" do |_simple_action, _parameter|
+    window.destroy
+    exit!
+  end
+
+  menubar = Gio::Menu.new()
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:38]: Could you omit parenthesis when no argument is needed?

```suggestion
  menubar = Gio::Menu.new
```

File: gtk4/sample/examples/menubar.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  menu_item_menu = Gio::MenuItem.new("Menu", nil)
+  menu = Gio::Menu.new()
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:40]:
```suggestion
  menu = Gio::Menu.new
```

File: gtk4/sample/examples/menubar.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  act_quit = Gio::SimpleAction.new("quit")
+  app.add_action(act_quit)
+  act_quit.signal_connect "activate" do |_simple_action, _parameter|
+    window.destroy
+    exit!
+  end
+
+  menubar = Gio::Menu.new()
+  menu_item_menu = Gio::MenuItem.new("Menu", nil)
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:39]: Can we omit `nil` here?

```suggestion
  menu_item_menu = Gio::MenuItem.new("Menu")
```

File: gtk4/sample/examples/menubar.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  menu = Gio::Menu.new()
+  menu_item_quit = Gio::MenuItem.new("Quit", "app.quit")
+  menu.append_item(menu_item_quit)
+  menu_item_menu.set_submenu(menu)
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:43]: Could you use assign style APi as much as possible for readability?

```suggestion
  menu_item_menu.submenu = menu
```

File: gtk4/sample/examples/menubar.rb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  menubar.append_item(menu_item_menu)
+  app.menubar = menubar
+  window.set_show_menubar true
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:46]:
```suggestion
  window.show_menubar = true
```

File: gtk4/sample/examples/menubar.rb [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+app.signal_connect "activate" do
+  window = Gtk::ApplicationWindow.new(app)
+  window.set_application(app)
+
+  act_quit = Gio::SimpleAction.new("quit")
+  app.add_action(act_quit)
+  act_quit.signal_connect "activate" do |_simple_action, _parameter|
+    window.destroy
+    exit!
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:35]: Do we need this?

```suggestion
```

File: gtk4/sample/examples/menubar.rb [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+require_relative "utils"
+
+require_gtk4
+
+app = Gtk::Application.new("org.gtk.example", :flags_none)
+
+app.signal_connect "activate" do
+  window = Gtk::ApplicationWindow.new(app)
+  window.set_application(app)
```
Comment (kou) [inline gtk4/sample/examples/menubar.rb:29]:
```suggestion
  window.application = app
```

**Comment by SamiSieranoja** [comment]
Do I need to do something more about this pull request?

**Comment by kou** [comment]
No. I'll merge this.

I just couldn't notice whether all review comments are responded or not without a new comment. Because I disable "new push on PR" notification. It's noisy to me.

**Pull request merged**